contracts/MintShop1155.sol

```diff
@@ -6,8 +6,12 @@ import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 
 import "./base/Sweepable.sol";
-import "./Super1155.sol";
-import "./Staker.sol";
+import "./interfaces/ISuper1155.sol";
+import "./interfaces/IStaker.sol";
+import "./interfaces/IMintShop.sol";
+
+
+import "./libraries/LibStorage.sol";
```

Comment:
1. This is necessary in order not to import contracts completely into the DropFactory contract. I put everything in a separate library, because it is effective by gas. 
2. I'm waiting for your suggestions :)
3. All of the structures that are placed in this library are necessary for correct operation.